plugins/package-managers/nuget/src/main/kotlin/utils/NuGetInspectorExtensions.kt

```diff
@@ -113,6 +113,7 @@ internal fun Collection<NuGetInspector.PackageData>.toOrtPackages(): Set<Package
 
         fun NuGetInspector.PackageData.getHash(): Hash =
             Hash.create(
+                @Suppress("UseOrEmpty")
```

Comment:
Have you considered supressing this for the whole function, so that readability of function body is not affected?

Reply:
Yes, I have considered this. In general I prefer to have `@Suppress` as narrow-scoped as possible, to not inadvertently suppress other occurrences (that might be added later to the code). I only lift `@Suppress` to a higher level if it make sense to suppress multiple clearly belonging statements at once.

Reply:
Ok, then let's leave it as is. Maybe we can collect other opinions in the dev meeting how they would see moving these to function level, but leave that out of scope here.

Reply:
Sure, feel free to put that topic onto the agenda 👍🏻